appenders/pythonAppender/README.md

```diff
@@ -1,5 +1,8 @@
 #Python custom log Handler
+- Install pythonAppender
+`pip install appenders/pythonAppender`
```

Comment:
Is this meant to be run from root or from the directory the README is in?

Reply:
it is meant to be ran from the root directory of